warehouse/query-core/src/main/java/datawave/query/config/ShardQueryConfiguration.java

```diff
@@ -54,174 +53,193 @@
+    protected boolean limitAnyFieldLookups = true; // TODO -- this is false by default in ShardQueryLogic
+    protected boolean bypassExecutabilityCheck = false;
+    /**
+     * Allows for back off of scanners.
+     */
+    protected boolean backoffEnabled = false;
+    /**
+     * Allows for the unsorted UIDs feature (see SortedUIDsRequiredVisitor)
+     */
+    protected boolean unsortedUIDsEnabled = true;
+    /**
+     * Allows us to serialize query iterator
+     */
+    protected boolean serializeQueryIterator = false;
+    /**
+     * Used to enable the SourceThreadTrackingIterator on the tservers
+     */
+    protected boolean debugMultithreadedSources = false;
+    /**
+     * Used to enable Event Field Value filtering in the TLD based on Query Expressions
+     */
+    protected boolean dataQueryExpressionFilterEnabled = false;
+    /**
+     * Used to enable sorting query ranges from most to least granular for queries which contain geowave fields in ThreadedRangeBundler
+     */
+    protected boolean sortGeoWaveQueryRanges = false;
+    /**
+     * Used to determine how many ranges the ThreadedRangeBundler should buffer before returning a range to the caller
+     */
+    protected int numRangesToBuffer = 0;
+    /**
+     * Used to determine how long to allow the ThreadedRangeBundler to buffer ranges before returning a range to the caller
+     */
+    protected long rangeBufferTimeoutMillis = 0;
+    /**
+     * Used to determine the poll interval when buffering ranges in ThreadedRangeBundler
+     */
+    protected long rangeBufferPollMillis = 100;
+    /**
+     * Used to determine the maximum number of query ranges to generate per tier when performing a geowave query.
+     */
+    protected int geoWaveMaxExpansion = 800;
+    /**
+     * Used to determine the maximum number of envelopes which can be used when generating ranges for a geowave query.
+     */
+    protected int geoWaveMaxEnvelopes = 4;
     private String shardTableName = "shard";
     private String indexTableName = "shardIndex";
     private String reverseIndexTableName = "shardReverseIndex";
     private String metadataTableName = "DatawaveMetadata";
     private String dateIndexTableName = "DateIndex";
-    private String indexStatsTableName = "indexStats";
-    
+    private String indexStatsTableName = "indexStats"; // TODO -- "shardIndexStats" from ShardQueryLogic
     private String defaultDateTypeName = "EVENT";
-    
-    // should we cleanup the shards and days hints that are sent to the
-    // tservers?
+    // should we cleanup the shards and days hints that are sent to the tservers?
     private boolean cleanupShardsAndDaysQueryHints = true;
-    
     // BatchScanner and query results options
-    private Integer numQueryThreads = 100;
-    private Integer numLookupThreads = 100;
-    private Integer numDateIndexThreads = 100;
+    private Integer numQueryThreads = 100; // TODO -- 8 in ShardQueryLogic
```

Comment:
I think in general the default used in the ShardQueryLogic is the one to use.  This is because the shard query logic version is copied into the configuration at the start.

Reply:
Got it.